native_client/BUILD

```diff
@@ -47,57 +47,62 @@ genrule(
+        "//tensorflow/core/kernels:constant_op",           # Const
         "//tensorflow/core/kernels:immutable_constant_op", # ImmutableConst
-        "//tensorflow/core/kernels:identity_op",        # Identity
-        "//tensorflow/core/kernels:transpose_op",       # Transpose
-        "//tensorflow/core/kernels:reshape_op",         # Reshape
-        "//tensorflow/core/kernels:shape_ops",          # Shape
-        "//tensorflow/core/kernels:strided_slice_op",   # StridedSlice
-        "//tensorflow/core/kernels:pack_op",            # Pack
-        "//tensorflow/core/kernels:reverse_op",         # ReverseV2
-        "//tensorflow/core/kernels:concat_op",          # ConcatV2
-        "//tensorflow/core/kernels:split_op",           # Split
-        "//tensorflow/core/kernels:sparse_to_dense_op", # SparseToDense
-        "//tensorflow/core/kernels:relu_op",            # Relu
-        "//tensorflow/core/kernels:bias_op",            # BiasAdd
-        "//tensorflow/core/kernels:math",               # Range, MatMul
-        "//tensorflow/core/kernels:tensor_array_ops",   # Placeholder, TensorArrayV3
-        "//tensorflow/core/kernels:control_flow_ops",   # Enter
-        "//tensorflow/core/kernels:ctc_ops",            # CTCBeamSearchDecoder
-        ### Needed by production model produced without "--nouse_seq_length
-        "//tensorflow/core/kernels:logging_ops",         # Assert
-        "//tensorflow/core/kernels:reverse_sequence_op", # ReverseSequence
-        # Classic deps
-        "//tensorflow/core/util/ctc",
-        "//third_party/eigen3",
+        "//tensorflow/core/kernels:identity_op",           # Identity
+        "//tensorflow/core/kernels:transpose_op",          # Transpose
+        "//tensorflow/core/kernels:reshape_op",            # Reshape
+        "//tensorflow/core/kernels:shape_ops",             # Shape
+        "//tensorflow/core/kernels:concat_op",             # ConcatV2
+        "//tensorflow/core/kernels:split_op",              # Split
+        "//tensorflow/core/kernels:variable_ops",          # VariableV2
+        "//tensorflow/core/kernels:relu_op",               # Relu
+        "//tensorflow/core/kernels:bias_op",               # BiasAdd
+        "//tensorflow/core/kernels:math",                  # Range, MatMul
+        "//tensorflow/core/kernels:control_flow_ops",      # Enter
+        #### Needed by production model produced without "--use_seq_length False"
+        #"//tensorflow/core/kernels:logging_ops",           # Assert
+        #"//tensorflow/core/kernels:reverse_sequence_op",   # ReverseSequence
     ] + if_native_model([
         "//tensorflow/compiler/tf2xla:xla_compiled_cpu_function",
-    ])
-      + if_cuda([
+    ])+ if_cuda([
         "//tensorflow/core:core",
-        "//tensorflow/core/kernels:slice_op_gpu",       # Slice GPU
     ]),
-    includes = ["kenlm", "boost_locale"],
+    includes = ["kenlm", "boost_locale", "c_speech_features", "kiss_fft130"],
```

Comment:
What about the `copts` tuning to disable `AVX`, `AVX2` and `FMA` for performances reasons?

Reply:
Good catch. I'll try to get bazel to do the right thing here.

Reply:
Maybe the reason why it was made is also not valid anymore?